<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndMain</class>
 <widget class="QMainWindow" name="WndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual graph editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/images/window-icon.png</normaloff>:/resource/images/window-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar
{
border: 0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.75, stop:0 rgb(160, 160, 160), stop:1 rgb(32, 32, 32));
border-top: 1px solid rgb(205, 205, 205, 192);
border-bottom: 1px solid rgb(205, 205, 205, 192);
border-width: 1px;
}

QToolBar::separator
{
background: rgb(205, 205, 205, 192);
width: 1px;
margin-left: 8px;
margin-right: 8px;
margin-top: 4px;
margin-bottom: 4px;
}

QToolButton
{
border: 0px;
color: rgb(205, 205, 205);
top: 0px;
left: 0px;
margin: 0px;
border-width: 0px;
padding: 1px;
margin-left: 4px;
margin-right: 4px;
}

QToolButton:hover
{
border: 0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.75, stop:0 rgb(225, 225, 225), stop:1 rgb(0, 0, 0));
border-right: 1px solid rgb(205, 205, 205, 192);
border-left: 1px solid rgb(205, 205, 205, 192);
color: white;
top: 0px;
left: 0px;
margin: 0px;
border-width: 1px;
padding: 0px;
margin-left: 4px;
margin-right: 4px;
}

QToolButton:pressed
{
border: 0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.75, stop:0 rgb(225, 225, 225), stop:1 rgb(64, 64, 64));
border-right: 1px solid rgb(205, 205, 205, 192);
border-left: 1px solid rgb(205, 205, 205, 192);
color: white;
top: 0px;
left: 0px;
margin: 1px;
border-width: 1px;
padding: 0px;
margin-left: 5px;
margin-right: 5px;
}

QToolButton:checked
{
border: 0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.75, stop:0 rgb(160, 225, 128), stop:1 rgb(48, 80, 32));
border-right: 1px solid white;
border-left: 1px solid white;
top: 0px;
left: 0px;
margin: 0px;
border-width: 1px;
padding: 0px;
margin-left: 4px;
margin-right: 4px;
}

QMenuBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(160, 160, 160), stop:1 rgb(32, 32, 32));
color: white;
margin: 0;
padding: 0;
}

QMenuBar:item
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(160, 160, 160), stop:1 rgb(32, 32, 32));
color: white;
}

QMenuBar:item:disabled
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(160, 160, 160), stop:1 rgb(32, 32, 32));
color: rgb(175, 175, 175);
}

QMenuBar:item:selected:disabled
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(160, 160, 160), stop:1 rgb(32, 32, 32));
color: rgb(175, 175, 175);
}

QMenuBar::item:selected
{
background: lightgray;
color: black;
margin: 0;
padding: 0;
}

QToolBox::tab
{
border-width: 1px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.75, y2:0, stop:1 rgb(96, 96, 96), stop:0 rgb(0, 0, 0));
border-bottom: 1px solid rgb(205, 205, 205, 192);
color:lightgray;
height:24px;
width:160px;
}

QToolBox::tab:hover
{
color: rgb(128, 255, 128);
}

QToolBox::tab::selected
{
color:white;
}

QStatusBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.75, y2:0, stop:1 rgb(96, 96, 96), stop:0 rgb(0, 0, 0));
color:lightgray;
border-bottom: 1px solid rgb(205, 205, 205, 192);
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabSpacing">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>439</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Top View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayoutTop">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="3" rowspan="2">
           <widget class="QScrollBar" name="vSBTop">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>32768</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QScrollBar" name="hSBTop">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>32768</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="GLDrawer2D" name="glDrawerTop" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>419</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Front View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayoutFront">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="3" rowspan="2">
           <widget class="QScrollBar" name="vSBFront">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>32768</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QScrollBar" name="hSBFront">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>32768</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="GLDrawer2D" name="glDrawerFront" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>419</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Side View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayoutSide">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="3" rowspan="2">
           <widget class="QScrollBar" name="vSBSide">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>32768</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QScrollBar" name="hSBSide">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>32768</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="GLDrawer2D" name="glDrawerSide" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>419</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Camera 3D View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="GLDrawer3D" name="glDrawer3D" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRenameVertex"/>
    <addaction name="separator"/>
    <addaction name="actionAutoconnect"/>
   </widget>
   <widget class="QMenu" name="menuGrid">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Grid</string>
    </property>
    <addaction name="actionAlignToGridMode"/>
    <addaction name="separator"/>
    <addaction name="actionAlignAllToGrid"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/resource/images/menu-languages.png</normaloff>:/resource/images/menu-languages.png</iconset>
     </property>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuGrid"/>
   <addaction name="menuGraph"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionDelete"/>
   <addaction name="actionAutoconnect"/>
   <addaction name="actionAlignToGridMode"/>
   <addaction name="separator"/>
   <addaction name="actionManual"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-new.png</normaloff>:/resource/images/menu-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-open.png</normaloff>:/resource/images/menu-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-save.png</normaloff>:/resource/images/menu-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-saveas.png</normaloff>:/resource/images/menu-saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-exit.png</normaloff>:/resource/images/menu-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-connect.png</normaloff>:/resource/images/menu-connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-disconnect.png</normaloff>:/resource/images/menu-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Alt+U</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-delete.png</normaloff>:/resource/images/menu-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionAutoconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-autoconnect.png</normaloff>:/resource/images/menu-autoconnect.png</iconset>
   </property>
   <property name="text">
    <string>Autoconnect</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAlignToGridMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-grid.png</normaloff>:/resource/images/menu-grid.png</iconset>
   </property>
   <property name="text">
    <string>Align To Grid Mode</string>
   </property>
  </action>
  <action name="actionAlignAllToGrid">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-aligntogrid.png</normaloff>:/resource/images/menu-aligntogrid.png</iconset>
   </property>
   <property name="text">
    <string>Align All To Grid</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/menu-help.png</normaloff>:/resource/images/menu-help.png</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/window-icon.png</normaloff>:/resource/images/window-icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/images/qt-logo.png</normaloff>:/resource/images/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English (English)</string>
   </property>
  </action>
  <action name="actionRenameVertex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename Vertex</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDrawer2D</class>
   <extends>QWidget</extends>
   <header>gldrawer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLDrawer3D</class>
   <extends>QWidget</extends>
   <header>gldrawer3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>WndMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
